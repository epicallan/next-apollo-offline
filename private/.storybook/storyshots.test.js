import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  configPath: 'private/.storybook'
});
