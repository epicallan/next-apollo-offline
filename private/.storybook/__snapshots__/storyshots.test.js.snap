// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Button With Icon 1`] = `
<button
  className="ui button"
  onClick={[Function]}
  tabIndex={undefined}
>
   
  <i
    aria-hidden="true"
    className="right arrow icon"
  />
   
</button>
`;

exports[`Storyshots Button With Icon and Text 1`] = `
<button
  className="ui button"
  onClick={[Function]}
  tabIndex={undefined}
>
   
  Continue 
  <i
    aria-hidden="true"
    className="right arrow icon"
  />
   
</button>
`;

exports[`Storyshots Button With Primary Color 1`] = `
<button
  className="ui primary button"
  onClick={[Function]}
  tabIndex={undefined}
>
   
  Primary
   
</button>
`;

exports[`Storyshots Button With Secondary Color 1`] = `
<button
  className="ui secondary button"
  onClick={[Function]}
  tabIndex={undefined}
>
   
  Secondary
   
</button>
`;

exports[`Storyshots Button With some emoji 1`] = `
<button
  className="ui button"
  onClick={[Function]}
  tabIndex={undefined}
>
   
  <span
    aria-label="emojis"
    role="img"
  >
    😀 😎 👍 💯
  </span>
   
</button>
`;

exports[`Storyshots Button with text 1`] = `
<button
  className="ui button"
  onClick={[Function]}
  tabIndex={undefined}
>
   
  Hello World
   
</button>
`;

exports[`Storyshots Menu default 1`] = `
<div
  className="ui menu"
/>
`;
